traffic_control/services/additional_sign.py

```diff
@@ -0,0 +1,66 @@
+from traffic_control.models import (
+    AdditionalSignPlan,
+    AdditionalSignPlanReplacement,
+    AdditionalSignReal,
+)
+from traffic_control.services.common import (
+    device_plan_create,
+    device_plan_get_active,
+    device_plan_get_current,
+    device_plan_replace,
+    device_plan_soft_delete,
+    device_plan_unreplace,
+    device_plan_update,
+)
+from users.models import User
+
+
+def additional_sign_plan_get_active():
+    return device_plan_get_active(AdditionalSignPlan)
+
+
+def additional_sign_plan_get_current():
+    return device_plan_get_current(AdditionalSignPlan)
+
+
+def additional_sign_plan_create(data: dict) -> AdditionalSignPlan:
+    return device_plan_create(
+        model=AdditionalSignPlan,
+        replace_method=additional_sign_plan_replace,
+        data=data,
+    )
+
+
+def additional_sign_plan_update(instance: AdditionalSignPlan, data: dict) -> AdditionalSignPlan:
+    return device_plan_update(
+        model=AdditionalSignPlan,
+        replace_method=additional_sign_plan_replace,
+        unreplace_method=additional_sign_plan_unreplace,
+        instance=instance,
+        data=data,
+    )
+
+
+def additional_sign_plan_replace(*, old: AdditionalSignPlan, new: AdditionalSignPlan):
+    device_plan_replace(
+        real_model=AdditionalSignReal,
```

Comment:
Ei ihan heti aukea, että miksi tässä on real_model=AdditionalSignReal mutta parametrit on AdditionalSignPlan

Reply:
Joo no tuo nyt tulee ilmi tuolta device_plan_replacesta, riittäis että tuonne services/common.py lisää docstringit funkkareille.

Reply:
Voin yrittää lisätä docstringiä joka avaa parametreja. Syy real_modelin käyttöön on että kun tehdään suunnitelman korvaus, niin vastaava toteuma täytyy muuttaa osoittamaan uutta suunnitelmaa.